raiden/tests/unit/storage/migrations/test_v18_to_v19.py

```diff
@@ -74,8 +74,8 @@ def test_upgrade_v18_to_v19(tmp_path):
     batch_query = storage.batch_query_state_changes(
         batch_size=500,
         filters=[
-            ('_type', 'raiden.transfer.state_change.ContractReceive%'),
-            ('_type', 'raiden.transfer.state_change.ActionInitChain'),
+            ('_type', '%raiden.transfer.state_change.ContractReceive%'),
+            ('_type', '%raiden.transfer.state_change.ActionInitChain%'),
```

Comment:
The percent signs in the beginning of the filters looks wrong, are you sure about this?

Reply:
I also think it's wrong. the type should always start with `raiden.X.Y`... so the % at the beginning can be removed.

Reply:
yeah it's wrong. I had it fixed before, but some rebase shenanigans reintroduced it...